### DISCUSSION TITLE
Easier way to delete an element?

### DISCUSSION THREAD

**Discussion by Ravbug** [mikke89/RmlUi#557] [Q&A] [answered]
I have the following RML
```html
<rml>
    <body>
        [some content cut for brevity]
        <label><input type="checkbox" id="showNav" value="1"></input>Show Navmesh</label>
    </body>
</rml>
```
In release mode, I want to delete the `showNav` element. Currently I'm doing it like this:
```cpp
 [[discard]] doc->RemoveChild(doc->GetElementById("showNav")->GetParentNode());
```
Is there an easier way to do this? In JavaScript one can do [`domElement.remove()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove).

**Comment by mikke89** [comment, marked as answer]
Yeah, that's the way to do it right now. I think it would make sense to add an `Element::Remove()` method. Pull request? :)
